modules/runtime/nuxeo-common/src/main/java/org/nuxeo/common/xmap/registry/Registry.java

```diff
@@ -32,6 +32,21 @@
      */
     boolean isNull();
 
+    /**
+     * Marks the registry with given identifier.
+     */
+    void mark(String id);
```

Comment:
Naming suggestion: `tag` or `label` instead of `mark`.

Reply:
Used "tag" instead